### PR TITLE
submitting project_one

### PR DESCRIPTION
**Pull request by tavarezjoel33**
Lehman Email: joel.tavarez1@lc.cuny.edu
General Email: tavarezjoel33@gmail.com

Extra Credit Options Completed: 
-Add CSS styling beyond what is required 
-Have 4 or more pages in your project (project 1 only)

### CODE REVIEW

**Comment by tavarezjoel33** [comment]
Lehman Email: joel.tavarez1@lc.cuny.edu
General Email: tavarezjoel33@gmail.com

Extra Credit Options Completed:
-Add CSS styling beyond what is required
-Have 4 or more pages in your project (project 1 only)

File: project_one/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+		<img src="images/lehman.png">
+
+		<h2 style="color: gold">Project Done by Joel Tavarez</h2>
+		<hr>
+
+
+	<h1 style="color: gold">This is Section One:</h1>
+	<p id="p1">This button will add text to the DOM.
+	<button id="grading-element-one" onclick="functionOne()" type="button"> Add a Thing</button>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+	</p>
+
+	<p id="p2"></p>
+
+	<hr>
+
+	<h1 style="color: gold">This is Section Two:</h1>
+	<p id="p3">This is an "About" Page that will redirect the webpage to a new page</p>
+	<a href="pages/about.html" id="grading-element-two"style="color: white">About</a>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+	<hr>
+
+	<h1 style="color: gold">This is Section Three:</h1>
+	<p>Below is a hidden text that will only appear when you press this button:</p>
+	<button onclick="functionTwo()" id="grading-element-three">Click me to Reveal the Truth</button>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+	<p style="visibility: hidden; color: white" id="grading-element-three-consequence">CSS and Javascript in one HTML is not the best way to make a webpage...duh</p>
+
+	<hr>
+
+	<h1 style="color: gold">This is Section Four:</h1>
+	<p>This is a form that will POST to the instructors LocalHost and return three truth values</p>
+
+	<form action="http://localhost:3000/form" method="POST" name="grading-element-four">
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

File: project_one/index.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+		<input type="submit" name="POST_to_Server">
+
+	</form>
+
+	<hr>
+
+	<h1 style="color: gold">This is Section Five</h1>
+	<p>This is a STRING when hovered over, it will turn GOLD:</p>
+
+	<p id="grading-element-five" onmouseover="functionThree()">DO NOT HOVER over this string, or else it will turn GOLD!</p>
```
Comment (lehmanclass) [inline project_one/index.html]: works as expected

**Comment by lehmanclass** [comment]
great! you have a 95, i need a little bit more added css rules before i can give the last 5.

**Pull request closed without merging**